src/components/Dashboard.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<DashboardWidget :items="notes"
+		:loading="loading"
+		:show-more-text="t('notes', 'notes')"
+		:show-more-url="showMoreUrl"
+	>
+		<template v-slot:default="{ item }">
+			<DashboardWidgetItem
+				:target-url="getItemTargetUrl(item)"
+				:main-text="item.title"
+				:sub-text="subtext(item)"
+				:item="item"
+			>
+				<template v-slot:avatar>
+					<div
+						class="note-item"
+						:class="{ 'note-item-favorite': item.favorite }"
+					/>
+				</template>
+			</DashboardWidgetItem>
+		</template>
+		<template v-slot:empty-content>
+			<EmptyContent icon="icon-notes">
+				<template #desc>
+					<p class="notes-empty-content-label">
+						{{ t('notes', 'No notes found') }}
+					</p>
+					<p>
+						<a :href="createNoteUrl" class="button">{{ t('note', 'Create note') }}</a>
```

Comment:
A bit better: "New note" to use the same wording as we do in the app itself for the button in the top left.

Reply:
Maybe `Add note`? I think in the Deck app it is called `Add card`, so maybe we can find a wording which fits to all those cases? (Maybe it's not that important though)

Reply:
I’d say it’s a bit different in both cases. There’s also e.g. Mail where you wouldn’t say "Add mail" or "Add message". :) For cards the "Add card" fits, but for notes the "New note" wording is a bit more fitting. Does that make sense?